### ISSUE TITLE
Showing related fields in a Select input

### ISSUE THREAD

**Issue by DriveSoft** [premieroctet/next-admin#132]
Showing related fields in a Select input

For example we have two tables Country and City, the City table has foreign key countryId, 
In _toString_ property we could specify a related field like: (city) => `${city.name} - ${city.country.name}`,

The same in the _formatter_ property.

options.tsx
```
....
model: {
      City: {
         title: "Cities",
         toString: (city) => `${city.name} (${city.country.name})`,
         list: {
            display: ["name", "country"],
            fields: {
               country: {
                  formatter: (country) => {
                     return <strong>{country?.name}</strong>;
                  },
               }
            }         
         },
         edit: {
            display: ["name", "latitude", "longitude", "country"]   
         }
      },
}
```

**Comment by cregourd** [comment]
Have you tried these options? Currently, it's possible, the type declaration file was designed to support related data access and Prisma data goes up to a related field. 
Also, since [v3.2.3](https://www.npmjs.com/package/@premieroctet/next-admin/v/3.2.3), the `optionsFormatter` option allows you to access this related data.

Tell me if I misunderstood your request and feel free to give feedback.

**Comment by DriveSoft** [comment]
Yes, I have tried. VSCode event suggests to me to choose the country table, but in Select component I see just for example:
New York (undefined)


      City: {
         title: "Cities",
         toString: (city) => `${city.name} (${city.country?.name})`,
         list: {
            display: ["name", "country"],
            fields: {
               country: {
                  formatter: (country) => {
                     return <strong>{country?.name}</strong>;
                  },
               }
            }         
         },
         edit: {
            display: ["name", "latitude", "longitude", "country"]   
         }
      },

![image](https://github.com/premieroctet/next-admin/assets/11293522/edd014e6-d4df-4d2c-a8aa-61c13c25d517)

**Comment by cregourd** [comment]
Issue fix available on [v3.2.5 ](https://www.npmjs.com/package/@premieroctet/next-admin/v/3.2.5) 🚀
So we close this issue, feel free to give feedback

**Comment by DriveSoft** [comment]
It works now, thanks!
